tests/protodata-extension/src/main/java/io/spine/protodata/test/annotation/AnnotationRenderer.java

```diff
@@ -72,9 +73,9 @@ private void renderFor(Annotated field, SourceFileSet sourceSet) {
             return;
         }
 
-        sourceSet.file(path)
-                 .at(getter)
-                 .withExtraIndentation(INDENT_LEVEL)
-                 .add('@' + field.getJavaAnnotation());
+
+        SourceFileKt.at(sourceSet.file(path), getter)
```

Comment:
See my PR-level comment on this one.

Reply:
Let's go for the second option.

Reply:
It's not about Java compatibility per se. It's about the language and readability when used in Java.